docs/reference/edot-sdks/android/configuration.md

```diff
@@ -0,0 +1,380 @@
+
+This is useful for setting dynamic global attributes. If you'd like to set static global attributes, which are also applied to metrics, take a look at [Intercept resources](#intercept-resources).
+
+```kotlin
+class MyApp : android.app.Application {
+
+    override fun onCreate() {
+        super.onCreate()
+        val agent = ElasticApmAgent.builder(this)
+            // ...
+            .addSpanAttributesInterceptor(interceptor)
+            .addLogRecordAttributesInterceptor(interceptor)
+            .build()
+    }
+}
+```
+
+### Session behavior
+
+You can configure how sessions work. For example:
+
+```kotlin
+class MyApp : android.app.Application {
+
+    override fun onCreate() {
+        super.onCreate()
+        val agent = ElasticApmAgent.builder(this)
+            .setSessionSampleRate(1.0) // <1>
+            // ...
+            .build()
+    }
+}
+```
+
+1. From version 1.1.0, you can provide your sample rate value. The value gets evaluated on every new session creation to determine whether the next session is sampled or not. When a session is not sampled, none of its signals are exported. Default value is `1.0`.
+
+### Disk buffering behavior
+
+You can configure how [disk buffering](index.md#disk-buffering) works. For example:
+
+```kotlin
+class MyApp : android.app.Application {
+
+    override fun onCreate() {
+        super.onCreate()
+        val agent = ElasticApmAgent.builder(this)
+            .setDiskBufferingConfiguration(DiskBufferingConfiguration.enabled()) // <1>
+            // ...
+            .build()
+    }
+}
+```
+
+1. From version 1.1.0, you can configure whether the disk-buffering feature is turned on or off. It's turned on by default.
+### Intercept resources
+
+The agent creates a [resource](https://opentelemetry.io/docs/specs/otel/overview/#resources) for your signals, which is a set of static global attributes. These attributes help {{kib}} properly display your application's data.
+
+You can intercept these resources and read or modify them as shown in the following example.
+
+```kotlin
+class MyApp : android.app.Application {
+
+    override fun onCreate() {
+        super.onCreate()
+        val agent = ElasticApmAgent.builder(this)
+            // ...
+            .setResourceInterceptor(interceptor)
+            .build()
+    }
+}
+```
+
+:::{note}
+The resource interceptor is only applied during initialization, as this is the only time where resource attributes can be modified. If you'd like to set _dynamic_ global attributes instead, take a look at [Intercept attributes](#intercept-attributes).
+:::
+
+### Intercept exporters
+
+The SDK configures exporters for each signal to manage features like [disk buffering](index.md#disk-buffering) and also to establish a connection with the Elastic export endpoint based on the provided [export connectivity](#export-connectivity).
```

Comment:
I've noticed that usually android developers aren't familiar with OTel (nor APM in some cases), which is why I tend to take every opportunity to provide them with informative links to help them understand what we're talking about across our docs, which is why for this paragraph I added links to the OTel signals descriptions in the [existing docs](https://www.elastic.co/docs/reference/apm/agents/android/configuration#intercepting-exporters). I know this might seem unnecessary for people like us, but you might be surprised about the amount of times we noticed that these simple terms aren't as widely known as we think. It's not a blocker though, we can leave it as is, but I think it would be nice if we added that extra info.